src/status_im2/common/bottom_sheet/styles.cljs

```diff
@@ -1,13 +1,14 @@
 (ns status-im2.common.bottom-sheet.styles
   (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
             [status-im.utils.platform :as platform]))
 
 (defn handle
   [override-theme]
   {:width            32
    :height           4
    :background-color (colors/theme-colors colors/neutral-100 colors/white override-theme)
-   :opacity          0.05
+   :opacity          (if theme/dark? 0.1 0.05)
```

Comment:
maybe it's better to use `theme-value` from "src/quo2/theme.cljs" as I think it will be easier to adjust that in the long run if needs be